js/runtime.js

```diff
@@ -1126,10 +1183,9 @@
                     synth.noteOff(midi, 100);
                   }
                 }).start();
-
                 alert("Play notes on the keyboard");
```

Comment:
Still not happy with having an alert here. Let's create another issue to deal with this better.